### PR TITLE
BAH Removing 4142 Transformation & Form from 526EZ

### PR DESCRIPTION
**Pull request by robfelder**
## Description
This PR covers the removal of:
- 4142 new choice page enhancements (including patient acknowledgment) from 526EZ
- Production flag from 526EZ flow
- Adjusting related unit tests
- Data transformation

## Testing done
Manual & Unit

## Acceptance criteria
- [x] 4142 form is not accessible
- [x] User still has the option to upload a document
- [x] In the event users do not have a document to upload, the user will be directed to download the form

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by igreulich — APPROVED** [review]

**Comment by robfelder** [comment]
@department-of-veterans-affairs/code-reviewers-for-external-teams-group wanted to put this on your radar as well for a review please.

**Review by markgreenburg — APPROVED** [review]
LGTM 🚀

File: src/applications/disability-benefits/526EZ/config/form.js [hunk 1]
```diff
@@ -742,65 +645,6 @@ const formConfig = {
             },
           },
         },
-        authorizationToDisclose: {
```
Comment (markgreenburg) [inline src/applications/disability-benefits/526EZ/config/form.js]: Thanks @robfelder !

**Review by cvalarida — APPROVED** [review]
Looks good. I think you may have to merge the latest master in, though? It looks like all the VA.gov builds are failing on this branch because of some missing links unrelated to this PR.

```
education/eligibility/index.html >>> href: "/education/advanced-training-and-certifications/", text: "Learn more about advanced training and certifications"
education/work-learn/non-traditional/index.html >>> href: "/education/work-learn", text: "
          Work learn
      "
```
image

**Comment by robfelder** [comment]
Thanks @cvalarida!

**Pull request merged**